### PR TITLE
ARC-1402 Update doc generation notebook

### PR DESCRIPTION
**Pull request by ryantimjohn**

### CODE REVIEW

**Comment by Frydafly** [comment]
> @Frydafly I've tested this a couple of times but forgot to take screenshots like a dummy. Let me know if you want me to do that or if you're okay testing yourself.

I haven't had a chance to test them! but I can by early next week -- if you think you'll get to it sooner, go ahead! otherwise, I'll get to it then.

**Comment by Frydafly** [comment]
hiya, testing this now

File: utils/utils/__init__.py [hunk 1]
```diff
@@ -7,3 +10,34 @@ def initialize_yaml():
...
+        click.echo(
+            "Could not find the profiles.yml file in the ~/.dbt/ directory.\
+            Please check that it exists.")
+        raise FileNotFoundError
+
+
+def run_dbt_subprocess(bash_command: str) -> str:
+    try:
+        process = subprocess.run(
+            bash_command, capture_output=True, shell=True, check=False)
```
Comment (Frydafly) [inline utils/utils/__init__.py:35]: just a note for now or for future that we could limit the scope of the subprocess by removing `shell=True` and setting the working directory with argument `cwd=WORKING_FOLDER_WITH_DBT_ARC_FUNCTION_MODELS` now that we have that constant at the start. 
reason being better security, less open-ended

I can't remember now if we use this function run_dbt_subprocess elsewhere, we probably do, and in that case, we'd have to modify that there too if we change this over.

**Review by Frydafly — APPROVED** [review]
Looks good to me, just one suggestion

File: utils/utils/__init__.py (hunk 1 above)
Comment (ryantimjohn) [reply utils/utils/__init__.py:35]: Yeah, this call is used in add_dependencies also, which doesn't have this variable: `WORKING_FOLDER_WITH_DBT_ARC_FUNCTION_MODELS`

**Pull request merged**